CTP/README.md

```diff
@@ -122,40 +124,65 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
   
 - **SHELL**
   - Prepare

```

Comment:
As is,
It's not required that you execute build for CTP
To be,
It's not required that you recompile CTP